### ISSUE TITLE
Document release calendar

### ISSUE THREAD

**Issue by ml019** [hamlet-io/docs#8]
the train/tram/unicycle release strategy needs to be documented (content was there in readthedocs) and linked to a calendar - preferrably something that dinasuarus offers out of the box but could be a static page.

**Comment by RossMurr4y** [comment]
When drop-down lists for the navbar become available (its on their issue list) then it would make sense to turn "releases" from a link into a heading, with "strategy" and "latest updates" as two options underneath.

For now I'll just add another page on the NavBar but I'm trying to keep it tidy so will keep this ticket open until the navbar drop-downs become available.

**Comment by ml019** [comment]
@RossMurr4y what's the ETA on the drop-down?

**Comment by RossMurr4y** [comment]
Well someone assigned the ticket 44 minutes ago...

**Comment by ml019** [comment]
@RossMurr4y sounds hopeful

**Comment by RossMurr4y** [comment]
The navbar update has been merged into the next docusaurus  release, just waiting on it to be published to NPM: https://github.com/facebook/docusaurus/milestone/3
